src/main/java/life/qbic/userdb/model/Person.java

```diff
@@ -0,0 +1,209 @@
+package life.qbic.userdb.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import life.qbic.datamodel.persons.Affiliation;
+import life.qbic.datamodel.persons.RoleAt;
+
+public class Person {
+  private int id;
+  private String firstName;
+  private String lastName;
+  private String email;
+  private int instituteID;
+  private String username;
+  private String title;
+  private String phone;
+  private Map<Integer, RoleAt> affiliationInfo; // ids and roles
+  private List<Affiliation> affiliations;
+
+  public Person(String username, String title, String first, String last, String eMail,
+      String phone, int affiliationID, String affiliationName, String affRole,
+      List<Affiliation> affiliations) {
+    super();
+    this.username = username;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = eMail;
+    this.phone = phone;
+    this.affiliations = affiliations;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    affiliationInfo.put(affiliationID, new RoleAt(affiliationName, affRole));
+  }
+
+  public Person(String username, String title, String first, String last, String eMail,
+      String phone, int affiliationID, String affiliationName, String affRole) {
+    super();
+    this.username = username;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = eMail;
+    this.phone = phone;
+    this.affiliations = new ArrayList<Affiliation>();
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    affiliationInfo.put(affiliationID, new RoleAt(affiliationName, affRole));
+  }
+
+  public Person(int id, String title, String first, String last, String email, String username, List<Affiliation> affiliations) {
+    this.id = id;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = email;
+    this.username = username;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    this.affiliations = affiliations;
+  }
+
+  public Person(String title, String first, String last, String email) {
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = email;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+  }
+
+  public Person(String zdvID, String firstName, String lastName, String email, String telephone,
+      int instituteID) {
+    super();
+    this.username = zdvID;
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.email = email;
+    this.phone = telephone;
+    this.instituteID = instituteID;
+  }
```

Comment:
To avoid this many constructors you could refactor it by introducing a builder pattern.